modules/framework/galasa-parent/galasa-demo-archetype/pom.xml

```diff
@@ -6,11 +6,11 @@
 	<parent>
 		<groupId>dev.galasa</groupId>
 		<artifactId>galasa-parent</artifactId>
-		<version>0.15.0</version>
+		<version>0.38.0</version>
 	</parent>
 	
 	<artifactId>galasa-demo-archetype</artifactId>
-	<version>0.15.0</version>
+	<version>0.38.0</version>
```

Comment:
I don't think this is used anywhere so it might be worth confirming then deleting so we don't release anything unnecessary in the 0.38.0 release - but that can be done after this PR is merged

Reply:
Raised separate story to look at this.